docs/css/webots-doc.css

```diff
@@ -894,7 +894,6 @@ body {
 
 .blog #view p {
   font-family: "Roboto";
-  width: 90%;
```

Comment:
Just fixed the bullet issues here. Actually, it was the paragraph which were too small. This is only visible on the new page, because the other blog pages do not contain bullet lists.